lib/matplotlib/tests/test_sphinxext.py

```diff
@@ -16,6 +16,17 @@
 tinypages = Path(__file__).parent / 'data/tinypages'
 
 
+def _ignore_build_artifacts(dir, files):
```

Comment:
From the docs, you can use `shutil.ignore_patterns` to generate this function directly.